Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
This is what is actually happening. Im not sure how this works, but CRUDController dont have the last parameter.

The example is the other way around, like @greg0ire said. If someone change my parent class (not sure if thats possible) it is considered BC break? This may be important to another PR I have in mind